packages/lakka/lakka-update/package.mk

```diff
@@ -36,6 +36,12 @@ make_target() {
   :
```

Comment:
Can you also remove `PKG_DEPENDS_TARGET="..."` (the whole line 25) - it does not need the `toolchain` to be built before the file is copied. Sorry, I did not see this before.

Reply:
Hi again! It seems I forgot to press the "Comment" button :laughing: 
I can fix that a bit later, but sure!